core/tee/tee_svc_cryp.c

```diff
@@ -19,7 +19,7 @@
 #include <utee_defines.h>
 #include <util.h>
 #if defined(CFG_CRYPTO_HKDF) || defined(CFG_CRYPTO_CONCAT_KDF) || \
-	defined(CFG_CRYPTO_PBKDF2)
+	defined(CFG_CRYPTO_PBKDF2) || defined(CFG_SECURE_KEY_STORAGE)
```

Comment:
I would prefer another `CFG_`. Maybe introduce `CFG_CRYPTO_RSASSA` to default enable in the related make script.
I even wonder if we could not simply drop the condition and always include **tee_api_defines_extensions.h**.